src/types/api/base.ts

```diff
@@ -29,3 +36,12 @@ export type Location = {
   kladr_code?: string;
   address: string;
 };
+
+export type Request = {
+  request_uuid?: string;
+  type: AddWebhookResponceType;
```

Comment:
`type` and `state` are properties of base type `Request`
Request is part of response of many api methods, not just `AddWebhook`
so here we need use alias name `RequestType`, or just define union here inplace (cause it will be used just here) like
```ts
type: "CREATE" | "UPDATE" ...
```
the same for state